lwb/metalang/cfg/cfg.spoofax2/syntax/part/language_base.sdf3

```diff
@@ -91,3 +92,11 @@ context-free syntax
   Part.ExportsSection = <exports {
   <{ExportsOption "\n"}*>
 }>
+
+context-free syntax
+
+  Part.EditorServicesSection = <editor-services {
```

Comment:
I'd rename this to a `reference-resolution` section, because it configures reference resolution. I'd like to move away from using Stratego for reference resolution in the future. Therefore, it would be good to set this up with variants in mind, with the only variant being Stratego-based reference resolution for now. Something like:

```
reference-resolution {
  variant = stratego {
    strategy = strategy editor-resolve
  }
}
```